about.html

```diff
@@ -12,53 +12,88 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
+    <header>
+        <div class=topOfAbout>
+        <section class='topOfAboutPage'>
+            <img src="img\lambda-black.png" alt="Lambda Icon;">
+            <nav class=topNavAbout>
+            <a href="">Home</a>
+            <a href="about.html">About</a>
+            </html>Products</a>
+            <a href="">Blog</a>
+            <a href="">Contact</a>
+            </nav>
+        </section>
+        <img src="img\jumbo-about.png" alt="firstImageAbout">
+        </div>
+        </header>
 
+    <div class="container about-page">
```

Comment:
This container could even go up above the header so it contains all of your content